radar-upload-backend/src/main/resources/dbChangelog.xml

```diff
@@ -65,7 +65,7 @@
             <column name="created_date" type="timestamp"/>
             <column name="file_name" type="varchar(255)"/>
             <column name="content_type" type="varchar(255)"/>
-            <column name="content" type="blob"/>
+            <column name="content" type="bytea"/>
```

Comment:
Can't you still use Blob data type even though you read it out as a blob? Alternatively you can still use a blob in the data structure, but read everything out as a byte array. Using blob gives us more flexibility to read the data out as a temporary file or stream it within the transaction at a later point if performance dictates that it should.